modules/hdf4_handler/hdfdesc.cc

```diff
@@ -5816,11 +5824,21 @@ void map_sds_var_dap4_attrs(HDFDMRArray_SDS *ar, int32 sds_id, int32 obj_ref, in
             throw InternalErr(__FILE__,__LINE__, "SDreadattr failed ");
             
         string dap4_attrname (attr_name);
+        if (emptySDS == 1) {
```

Comment:
I see in sonarscan that it thinks these three if statements can be combined into one. Is that true? If so, it might make this easier to understand. A minor issue, so change this at your discretion.

Reply:
I noticed this. Actually I deliberately do this way to make the logic clear since the "nested ifs" follow the logic in this case.